features/lwipstack/mbed_lib.json

```diff
@@ -128,6 +128,10 @@
         "ppp-thread-stacksize": {
             "help": "Thread stack size for PPP (obsolete: use netsocket/ppp configuration instead)",
             "value": 768
+        },
+        "rawipsocket-enabled": {
+            "help": "Enable ICMP RAW",
```

Comment:
Could you adjust this to "Enable lwip raw sockets; required for Mbed OS ICMPSocket)"? Or something like that?

Reply:
Oh, and you've lost the last set of changes - that should be called `raw-socket-enabled`, and the code is expecting that.

Reply:
OK let me rewind that one